app-backend/db/src/test/scala/com/azavea/rf/database/OrganizationDaoSpec.scala

```diff
@@ -1,30 +1,85 @@
 package com.azavea.rf.database
 
 import com.azavea.rf.datamodel.Organization
+import com.azavea.rf.datamodel.Generators.Implicits._
 import com.azavea.rf.database.Implicits._
 
 import doobie._, doobie.implicits._
 import cats._, cats.data._, cats.effect.IO
 import cats.syntax.either._
 import doobie.postgres._, doobie.postgres.implicits._
 import doobie.scalatest.imports._
+import org.scalacheck.Prop.forAll
 import org.scalatest._
+import org.scalatest.prop.Checkers
 
 
-class OrganizationDaoSpec extends FunSuite with Matchers with IOChecker with DBTestConfig {
+class OrganizationDaoSpec extends FunSuite with Matchers with Checkers with DBTestConfig {
 
-  test("insertion") {
-    val testName = "test org"
+  // create
+  test("insert an organization from a name") {
+    check {
+      forAll(
+        (name: String) => {
+          val withoutNull = name.filter( _ != '\u0000' ).mkString
```

Comment:
should we check the inverse?

Reply:
I'm not sure -- postgres hates the `null` byte and doobie throws `PSQLException`s if you try to insert anything with a `null` byte in it. I included this as a concession to the fact that we don't have a codified attitude toward the `null` byte right now, and I'm not going to fix that in our database interactions as part of tests.